lib/travis/api/build/app.rb

```diff
@@ -54,27 +74,30 @@ class App < Sinatra::Base
 
           if ENV['SENTRY_DSN']
             Raven.extra_context(
-              repository: payload['repository']['slug'],
-              job: payload['job']['id'],
+              repository: payload.fetch('repository', {}).fetch('slug', '???'),
+              job: payload.fetch('job', {}).fetch('id', '???'),
             )
           end
 
-          content_type :txt
-          Travis::Build.script(payload).compile
-        end
+          compiled = Travis::Build.script(payload).compile
 
-        get '/uptime' do
-          status 204
+          content_type 'application/x-sh'
+          status 201
```

Comment:
Is 201 the best status code for this? No resource is being "created", travis-build mostly just "computes" one. For 201s I would expect I could then hit some endpoint to retrieve the created resource.